app/pages/api/auth/lnurl/generate-secret.ts

```diff
@@ -42,7 +42,8 @@ export default async function handler(
     tag: "login",
   });
 
-  const callbackUrl = `https://${
+  const protocol = process.env.LNURL_CALLBACK_PROTOCOL || 'https:';
+  const callbackUrl = `${protocol}//${
```

Comment:
thanks for looking into this!

I am wondering if this can be simplified - does it need a new env variable?

I think we can replace this and just use:

`const callbackUrl = `${process.env.APP_URL}/api/auth/lnurl/do-login?${params.toString()}`;`

it seems to work locally for me.

Reply:
Yes that's better.